LICENSE.md

```diff
@@ -0,0 +1,27 @@
+# W3C SOFTWARE AND DOCUMENT NOTICE AND LICENSE
+
+http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document
+
+This work is being provided by the copyright holders under the following license.
+
+License
+-------
+
+By obtaining and/or copying this work, you (the licensee) agree that you have read, understood, and will comply with the following terms and conditions.
+
+Permission to copy, modify, and distribute this work, with or without modification, for any purpose and without fee or royalty is hereby granted, provided that you include the following on ALL copies of the work or portions thereof, including modifications:
+
+The full text of this NOTICE in a location viewable to users of the redistributed or derivative work.
+
+Any pre-existing intellectual property disclaimers, notices, or terms and conditions. If none exist, the W3C Software and Document Short Notice should be included.
+
+Notice of any changes or modifications, through a copyright statement on the new code or document such as "This software or document includes material copied from or derived from [title and URI of the W3C document]. Copyright © [YEAR] W3C® (MIT, ERCIM, Keio, Beihang)."
```

Comment:
Thank you. Updated. You can edit online from https://github.com/rbouqueau/TTML_in_MP4_DASH_statement/edit/license/LICENSE.md